CHANGELOG.md

```diff
@@ -4,6 +4,39 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
+
+## 2.17.0 (2024-09-20)
+
+This release contains performance improvements and bug fixes since
+the 2.16.1 release. We recommend that you upgrade at the next
+available opportunity.
+
+
+**Features**
+* #6882: Allow DELETE on the compressed chunks without decompression.
+* #7033: Use MERGE statement on Continuous Aggregates Refresh.
```

Comment:
could you please use lower case here, we're trying to use it consistently for feature names

Reply:
edited so.